object.js

```diff
@@ -0,0 +1,69 @@
+const movie = {
+  title: "johan whake",
+  year: 2001,
+  director: "cristofer",
+  actors: "jamse ",
+  rating: 8.5,
+  timesWatched: 2,
+
+  recommend() {
+    console.log("this film its amaizn");
+  },
+
+  watch() {
+    this.timesWatched = this.timesWatched + 1;
+    console.log(this.timesWatched);
+  },
+  setRating(newRating) {
+    this.rating = newRating;
+  },
+  addActor(actorName) {
+    this.actors += actorName;
```

Comment:
This one is working fine, but you could have used better suggestions like the concat method or the plus sign:
- `this.actors = this.actors + ", " + actorName;`
- this.actors = this.actors.concat(", ", actorName);